### ISSUE TITLE
Why not provider a clean project source code?

### ISSUE THREAD

**Issue by foryoung2018** [pelya/xserver-xsdl#174]
So many no useful code prevent me to make this project to be a wide, intresting, useful.
Can you sir refactor these native c code?
Even though just delete no use code, compile file will very helpful for me.
Many many thanks!!!

**Comment by foryoung2018** [comment]
search word, git track file is so diffcult.
fix bug, add new feature, implement extention is so diffcult.
:(

**Comment by pelya** [comment]
No. You've got what you've got, an unmaintained codebase from 1991. Use git
diff with the upstream branch to get Android-specific changes.

**Comment by foryoung2018** [comment]
OK, if I want to develop a new xserver on Android as xserveer-xsdl，can you help to give any useful refer code/design/doc ?

**Comment by foryoung2018** [comment]
https://github.com/pelya/xserver-xsdl/commit/e6cef9e7aa14228aad1e4551570a1823c3c1e46c
@pelya  start with this commit in branch xsdl2?

**Comment by pelya** [comment]
Yes
